bookmarks.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+
+  <body>
+    <header class="header">
+      <h1>QUIZ APP</h1>
+    </header>
+
+    <main>
+      <!-- QUESTION CARDS -->
+
+      <section class="card">
+        <h2 class="card__heading">What property flips the axes in flexbox?</h2>
+        <button class="card__button-bookmark" aria-label="Save bookmark">
+          📕
+        </button>
+        <button class="card__button-answer" aria-label="Show Answer">
+          Show Answer
+        </button>
+
+        <span class="card__tag">
+          <i class="tag">&num;html</i>
+          <i class="tag">&num;flexbox</i>
+          <i class="tag">&num;css</i>
+        </span>
+      </section>
+
+      <section class="card">
+        <h2 class="card__heading">Lorem Ipsum dolor sit?</h2>
+        <button class="card__button-bookmark" aria-label="Save bookmark">
+          📕
+        </button>
+        <button class="card__button-answer" aria-label="Show Answer">
+          Show Answer
+        </button>
+
+        <span class="card__tag">
+          <i class="tag">&num;html</i>
+          <i class="tag">&num;flexbox</i>
+          <i class="tag">&num;css</i>
+        </span>
+      </section>
+    </main>
+  </body>
+
+  <nav>
```

Comment:
nav muss in body sein, du kannst  im body footer erstellen und da dein nav einfügen